components/more-less/more-less.js

```diff
@@ -120,9 +121,11 @@ class MoreLess extends LocalizeMixin(LitElement)  {
 		this.__blurBackground = 'linear-gradient(rgba(255, 255, 255, 0) 0%, rgb(255, 255, 255) 100%)';
 		this.__transitionAdded = false;
 
+		this.expanded = false;
```

Comment:
Should something toggle this to `true` when "more" is shown?

Reply:
Yup that happens in here https://github.com/BrightspaceUI/core/blob/master/components/more-less/more-less.js#L282

Reply:
Oh sorry I missed that it wasn't a new property!